unified-codes/apps/web/src/types/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from './Alert';
```

Comment:
Noticed the Alert stuff isn't used anywhere except the front-end, so moved it into the web app!

Reply:
ah! I wondered about the possible offroading..

Reply:
![](https://media4.giphy.com/media/ekMiPZzWe5ion9m6nG/giphy-downsized-medium.gif)